pkg/ccr/base/spec.go

```diff
@@ -1507,6 +1507,24 @@ func (s *Spec) DesyncTables(tables ...string) error {
 	return nil
 }
 
+func (s *Spec) ModifyTableProperty(destTableName string, modifyProperty *record.ModifyTableProperty) error {
+	dbName := utils.FormatKeywordName(s.Database)
+	destTableName = utils.FormatKeywordName(destTableName)
+	modifySql := modifyProperty.Sql
+	modifySql = strings.ReplaceAll(modifySql, "\u003d", "=")
+
+	validProperties := CheckModifyTablePropertySql(modifyProperty.Properties)
+	if len(validProperties) != 0 {
+		log.Warnf("don't support alter table properties: %s", strings.Join(validProperties, ","))
+		return nil
+	}
```

Comment:
How about only syncing the supported properties and filtering the unsupported properties directly? If there are no supported properties after filtering, return nil.